src/test/java/hudson/plugins/parameterizedtrigger/test/PredefinedPropertiesBuildTriggerConfigTest.java

```diff
@@ -2,6 +2,7 @@
  * The MIT License
  *
  * Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi
+ * Copyright (c) 2018 Motorola Solutions, Inc.
```

Comment:
I'm not too comfortable with adding a copyright for a company for an OSS plugin.  I'm fine with even just removing the Sun copyright line as well.

Reply:
Thank you for the review. Unfortunately our internal requirements oblige me to put this in to be allowed to submit the change back to the upstream. However it only requires to do so if there is already a copyright, so I believe if all the copyrights are removed altogether then it would be fine as well.